packages/angular/README.md

```diff
@@ -1,28 +1,146 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
 
 ### Complex Forms in the blink of an eye
 
-JSONForms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+JSON Forms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
 
-# Angular Package
+## Angular Package
 
-This is the JSONForms Angular package which provides the necessary bindings for Angular. It uses [JSONForms Core](https://www.npmjs.com/package/@jsonforms/core).
+This is the JSON Forms Angular package which provides the necessary bindings for Angular. It uses [JSON Forms Core](https://www.npmjs.com/package/@jsonforms/core).
 
-You can combine the Angular package with any Angular-based renderer set you want, for example the [Material Renderers](https://github.com/eclipsesource/jsonforms/blob/master/packages/angular-material).
+You can combine the Angular package with any Angular-based renderer set you want, for example the [Material Renderers](https://www.npmjs.com/package/@jsonforms/angular-material).
 
```

Comment:
Also mention and link the Angular seed and the Angular documentation on JSON forms website.